hooks/pre-commit

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -e
```

Comment:
`exec 1>&2` is a better practice that redirects stdout to stderr in hooks because some of users may wanna use stdout of `git commit`.

```suggestion
set -e

exec 1>&2
```